drupal/templates/drupal-cron.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   suspend: false
   jobTemplate:
     spec:
-      backoffLimit: 1
+      backoffLimit: {{- default 1 $job.backoffLimit}}
```

Comment:
Instead of using the `default` function here, please move the default value of `1` to the values.yaml file. Also, please add a brief comment there (in values.yaml) explaining what backofflimit does, so developers know what to use this setting for.

Reply:
`default` needs to be used anyway as there might be custom cronjobs and that value can be skipped for those. I added value for default cron `drush cron` in values.yml and some comment.